assets/js/src/ga-integration.js

```diff
@@ -0,0 +1,15 @@
+// eslint-disable-next-line camelcase
+const google_analytics_integration_product_data = [];
```

Comment:
Interesting. I added `window.` in the creation of the variable and also use object instead of array. 

https://github.com/woocommerce/woocommerce-google-analytics-integration/pull/261/commits/be572242209a248a42db9fd8b5b1b66c5d443f2d